tests/test_flaskparser.py

```diff
@@ -33,23 +33,31 @@ def test_use_args_with_view_args_parsing(self, testapp):
         assert res.json == {"view_arg": 42}
 
     def test_use_args_on_a_method_view(self, testapp):
-        res = testapp.post("/echo_method_view_use_args", {"val": 42})
+        res = testapp.post_json("/echo_method_view_use_args", {"val": 42})
         assert res.json == {"val": 42}
 
     def test_use_kwargs_on_a_method_view(self, testapp):
-        res = testapp.post("/echo_method_view_use_kwargs", {"val": 42})
+        res = testapp.post_json("/echo_method_view_use_kwargs", {"val": 42})
         assert res.json == {"val": 42}
 
     def test_use_kwargs_with_missing_data(self, testapp):
-        res = testapp.post("/echo_use_kwargs_missing", {"username": "foo"})
+        res = testapp.post_json("/echo_use_kwargs_missing", {"username": "foo"})
         assert res.json == {"username": "foo"}
 
     # regression test for https://github.com/marshmallow-code/webargs/issues/145
     def test_nested_many_with_data_key(self, testapp):
-        res = testapp.post_json("/echo_nested_many_data_key", {"x_field": [{"id": 42}]})
-        # https://github.com/marshmallow-code/marshmallow/pull/714
+        post_with_raw_fieldname_args = (
+            "/echo_nested_many_data_key",
+            {"x_field": [{"id": 42}]},
+        )
+        # under marhsmallow2 this is allowed and works
```

Comment:
```suggestion
        # under marshmallow 2 this is allowed and works
```

Reply:
This was done in 176a9f7 (and I marked co-authorship the way github does out of curiosity, to see if it would understand and resolve this suggestion).
It doesn't seem to have, so I'm just going to mark the accepted suggestions as resolved.